arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -560,9 +560,343 @@
+          },
+          "MaxDeliveryCount ": {
+           "format": "int32",
+           "type": "integer",
+           "description": "Gets or sets the maximum delivery count. A message is automatically deadlettered after this number of deliveries."
+          },
+          "MaxSizeInMegabytes ": {
+           "format": "int64",
+           "type": "integer",
+           "description": "Gets or sets the maximum size of the queue in megabytes, which is the size of memory allocated for the queue."
+          },
+           "MessageCount ": {
+           "format": "int64",
+           "type": "integer",
+           "description": "Gets the number of messages in the queue."
+          },
+          "MessageCountDetails": {
+          "type": "object",
+          "description": "Gets message details for a queue."
+        },
+        "Path": {
+          "type": "string",
+          "description": "Gets or sets the name of the queue."
+        },
+         "RequiresDuplicateDetection": {
+          "type": "boolean",
+          "description": "Gets or sets the value indicating if this queue requires duplicate detection."
+        },
+         "RequiresSession": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether the queue supports the concept of session."
+        },
+        "SizeInBytes ": {
+           "format": "int64",
+           "type": "integer",
+           "description": "Gets the size of the queue in bytes."
+          },
+         "EntityStatus ": {
+          "type": "string",
+          "description": "Enumerates the possible values for the status of a messaging entity.",
+          "enum": [
+            "Active",
+            "Creating",
+            "Deleting",
+            "Disabled",
+            "ReceiveDisabled",
+            "Renaming",
+            "Restoring",
+            "SendDisabled",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "EntityStatus",
+            "modelAsString": false
+          }
+        },
+        "SupportOrdering": {
+          "type": "boolean",
+          "description": "Gets or sets a value that indicates whether the queue supports ordering."
+        },
+        "UpdatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets the exact time the message has been updated."
+        },
+         "UserMetadata": {
+          "type": "string",
+          "description": "Gets or sets the user metadata."
+        }
+    },
+      "description": "Represents the metadata description of the queue."
+    },
+    "QueueResource": {
+      "properties": {
+                   "properties": {
+                "$ref": "#/definitions/QueueProperties",
+                "description": "Gets or sets properties of the Queues."
```

Comment:
`description` should be placed in the definition for `#/definitions/QueueProperties`